README.md

```diff
@@ -215,3 +215,59 @@ curl 'http://localhost:5001/api/v0/object/get?arg=QmYEqnfCZp7a39Gxrgyv3qRS4MoCTG
 #### Pin
 
 #### Gateway
+
+#### Files
+
+##### mkdir
+
+```JavaScript
+ipfs.files.mkdir(<folderName>, function (err) {})
+```
+
+##### cp
+
+```JavaScript
+ipfs.files.cp([<pathSrc>, <pathDst>], function (err) {})
+```
+
+##### ls
+
+```JavaScript
+ipfs.files.ls(<path>, function (err, res) {})
+```
+
+##### stat
+
+```JavaScript
+ipfs.files.stat(<path>, function (err, res) {})
+```
+
+##### rm
+
+```JavaScript
+ipfs.files.rm(<path>, [<options>],  function (err) {})
+```
+
+For `rm -r` pass a options obj with `r: true`
```

Comment:
It would be noce to have `recursive: true` as an alias for `r: true` as that is much more readable in code segment. Also is this the only option that can be passed atm?